### DISCUSSION TITLE
really unfortunate water meter - is there even a chance of recognizing these analog dials?

### DISCUSSION THREAD

**Discussion by kongomongo** [jomjol/AI-on-the-edge-device#3087] [Q&A] [answered]
Hi everyone,

thanks for this awesome piece of software!

Unfortunately I have a rather special water meter, and my ana3+4 dials seem odd. I tried adding them but totally wrong values were detected, that's why I have removed them for now.

ana4 has a metal cover turning opposite the arrow.
ana3 has a irregularity in the cover glass and is partially hidden.

any advise whether those are detectable or if theres no chance?

Thanks!

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/5646390/fc0cff5a-1c93-4e16-a427-fdfa459a690c)

**Comment by caco3** [comment, marked as answer]
ana4 should work ok, I think there are already training images in the models for this.
For ana3 I think you would need to train your own model. How ever it might be easier to just keep ana3 and ana4 away. When using https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#PostProcessing-NUMBER.ExtendedResolution you could almost get to the value of ana3.

**Comment by kongomongo** [comment]
yep that seems to work! thank you!
